content/upbound-marketplace/providers.md

```diff
@@ -2,57 +2,100 @@
 title: "Official Providers"
 weight: 20
 ---
-Upbound creates, maintains and supports a set of Crossplane providers called *official providers*.  
+Upbound creates, maintains and supports a set of Crossplane providers called **official providers**.  
 
-## Find official providers
-Identify official providers in the marketplace with the `by Upbound` gold seal.
+Upbound builds, tests and supports official providers. The Upbound official providers are open source under the Apache 2.0 license. Upbound recommends the official providers for all deployments.
 
-{{<img src="upbound-marketplace/images/provider-by-upbound.png" alt="example official provider with a by Upbound seal" size="xtiny">}}
+Find official providers in the [Upbound Marketplace](https://marketplace.upbound.io/providers?tier=official). 
+All official providers have a purple `Official` tag.
 
-Also find official providers by filtering a [search in the marketplace](https://marketplace.upbound.io/providers?tier=official) to just `Official`.
+{{<img src="/upbound-marketplace/images/provider-by-upbound.png" alt="Official provider tag" >}}
 
-{{<img src="upbound-marketplace/images/official-provider-search-filter.png" alt="a marketplace search filter with Providers and Official filters set" size="xtiny" >}}
+## Provider families
+Upbound packages the official providers for Amazon AWS, Google GCP and Microsoft Azure as a "family." 
```

Comment:
```suggestion
Upbound packages each of the official providers for Amazon AWS, Google GCP and Microsoft Azure as a "family." 
```

Trying to avoid it sounding like AWS, GCP and Azure together make a family.